src/datetime/mod.rs

```diff
@@ -174,14 +174,19 @@ pub enum SecondsFormat {
 /// the time.
 ///
 /// Timezone-aware calculations on a [`DateTime`] can be done with:
-/// - [`Sub<DateTime>`] to get a [`Duration`](struct.Duration.html) with the difference between two
-///   datetimes.
-/// - [`Add<Duration>`] to add a [`Duration`](struct.Duration.html) of an absolute number of
+/// - [`signed_duration_since`](#method.signed_duration_since) to get a [`Duration`](
```

Comment:
I think we should mention both the operator and the checked methods here.

Reply:
The line below says "The same functionality is also available with implementations of `Add` and `Sub`."
I initially made those more prominent, but they are not a great idea to use because they can panic in pretty common scenarios.

To make them a bit more visible I added an example of working with `DateTimes`.